heptagram-devs.html

```diff
@@ -40,9 +40,15 @@ <h1>Developers</h1>
         <div class="head-developer">
             <img class="avatar" src="img/midou's avatar.png" alt="midou's avatar">
```

Comment:
One question: can't we take the images from GitHub instead of saving them locally?

Reply:
We actually should do that. @MidouWebDev.... Want to do this? Or have @Panquesito7 work on it?

Reply:
> We actually should do that. @MidouWebDev.... Want to do this? Or have @Panquesito7 work on it?

I would like to work on it, but if @MidouWebDev wants to, feel free to work on that. 🙂

Reply:
It is up to him

Reply:
@Panquesito7 you can do it, bro !

Reply:
Thanks! Working on it. 😄